### PR TITLE
Chapter Six Challenge one intro pages

### LINKED ISSUE #765: Chapter 6 Transition Pages
Add the transition pages for Chapter 6, this includes implementing the new transaction overview image/object more information is needed on the scope for this

image

https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=6398%3A17385&mode=dev

https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=6401%3A18420&mode=dev

### LINKED ISSUE #766: Chapter 6 Code introduction Page
Implement the Page that introduces a transaction block

image

https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=6398%3A17173&mode=dev

### PR DESCRIPTION
**Pull request by 0tuedon**
This PR closes #765 
Designed both intro pages for challenge one in Chapter six

### CODE REVIEW

**Comment by 0tuedon** [comment]
Oops guess the intro one should be this issue  #766  making this a draft to change

**Comment by 0tuedon** [comment]
> Oops guess the intro one should be this issue #766 making this a draft to change

Ready for review

File: i18n/locales/en.ts [hunk 1]
```diff
@@ -1063,8 +1063,22 @@ const translations = {
     },
     paragraph_one:
       'Vanderpoole is out to get you now that you’ve exposed him as a fraud. But despite that, plenty of scared people continue to cling to the myth he created around himself, his family, and their supposed ancestry. Times are scary, and people need a hero. Unfortunately, for many, he’s the best they’ve got.',
-    in_out_one: {
+    in_out_two: {
       title: 'The ins and outs',
```
Comment (benalleng) [inline i18n/locales/en.ts:1067]: You'll need to adjust the metadata titles to match this now being under in_out_two

File: public/assets/images/chapter-6-in-out-2.jpg
Comment (benalleng) [inline public/assets/images/chapter-6-in-out-2.jpg:1]: Let's export the image a little differently as this one seems to include its own border

**Review by benalleng — CHANGES_REQUESTED** [review]
Looks like you swapped all the file names around

File: public/assets/images/chapter-6-in-out-2.jpg
Comment (0tuedon) [reply public/assets/images/chapter-6-in-out-2.jpg:1]: seems it's still the same?

**Review by benalleng — CHANGES_REQUESTED** [review]
few small changes but looking good!

**Review by benalleng — APPROVED** [review]
LGTM!

**Pull request merged**